action.yml

```diff
@@ -337,12 +340,14 @@ runs:
   steps:
     - id: lsb-release
       run: |
-        source /etc/lsb-release
-        echo "id=${DISTRIB_ID}" >> $GITHUB_OUTPUT
-        echo "release=${DISTRIB_RELEASE}" >> $GITHUB_OUTPUT
-        echo "codename=${DISTRIB_CODENAME}" >> $GITHUB_OUTPUT
-        echo "description=${DISTRIB_DESCRIPTION}" >> $GITHUB_OUTPUT
-        echo "id-release=${DISTRIB_ID}-${DISTRIB_DESCRIPTION}" >> $GITHUB_OUTPUT
+        if [ "$RUNNER_OS" == "Linux" ]; then
+          source /etc/lsb-release
+          echo "id=${DISTRIB_ID}" >> $GITHUB_OUTPUT
+          echo "release=${DISTRIB_RELEASE}" >> $GITHUB_OUTPUT
+          echo "codename=${DISTRIB_CODENAME}" >> $GITHUB_OUTPUT
+          echo "description=${DISTRIB_DESCRIPTION}" >> $GITHUB_OUTPUT
+          echo "id-release=${DISTRIB_ID}-${DISTRIB_DESCRIPTION}" >> $GITHUB_OUTPUT
```

Comment:
We are using `id-release` in the cache key below. Should we set this for MacOS/Darwin too?

Reply:
Good catch - is set now.